boards/xtensa/intel_adsp_cavs15/intel_adsp_cavs15.yaml

```diff
@@ -4,3 +4,7 @@ type: mcu
 arch: xtensa
 toolchain:
   - zephyr
+testing:
+  only_tags:
+     - kernel
```

Comment:
this was done for others to limit duplication, we still need to run other tests at least on one platform, otherwise our coverage will be significantly reduced. I would rather see the coverage increase on other platforms instead of reducing it here.

Reply:
Maybe merge this first as a baseline then work to add coverage?  FWIW: cavs15 is approaching legacy status now anyway and all development is happening elsewhere.  If we want to expand the test set we want to be doing it on TGL.